### PR TITLE
yoo

### PR DESCRIPTION
**Pull request by felipelincoln**
XD

### CODE REVIEW

File: dd [hunk 1]
```diff
@@ -1 +1,6 @@
 d
+ddtank
+dinosaur
+dynacom
+dynamics
```
Comment (learninglearninglearning) [inline dd]: where did those came from?

Comment (felipelincoln) [reply dd]: got it!

**Comment by learninglearninglearning** [comment]
ok, let's merge it!

**Comment by felipelincoln** [comment]
good shit

**Pull request merged**